### PR TITLE
Update proc_inspect-slsa-provenance.adoc

### PR DESCRIPTION
**Pull request by Roming22**
Add details about the Tekton Chains public-key.
The information is needed for users to verify the SLSA provenance

### CODE REVIEW

**Review by ralphbean — APPROVED** [review]

**Comment by Roming22** [comment]
Tagging @Chr1st1anSears since they seem to be in charge of merge :wink:

**Review by Chr1st1anSears — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_inspect-slsa-provenance.adoc [hunk 1]
```diff
@@ -137,5 +137,5 @@ Example output:
 
 == Additional resources
 * Learn about the SLSA framework and xref:../concepts/slsa/con_slsa-conformity.adoc[how {ProductName} meets the requirements of SLSA Build Level 3].
-* Red Hat's Enterprise Contract (EC) is a powerful tool that you can also use to verify your SLSA provenance; visit link:https://enterprisecontract.dev/posts/introducing-the-enterprise-contract/[this page]  to learn how to use the EC CLI tool to verify your provenance.
+* Red Hat's Enterprise Contract (EC) is a powerful tool that you can also use to verify your SLSA provenance; visit link:https://enterprisecontract.dev/posts/introducing-the-enterprise-contract/[this page]  to learn how to use the EC CLI tool to verify your provenance. You will need the public key used by Tekton Chains, which available as the *public-key* secret in the *openshift-pipelines* namespace and readable to all authenticated users.
```
[the remark this answers is not in the document]
Comment (Roming22) [reply docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_inspect-slsa-provenance.adoc]: Done

**Comment by Roming22** [comment]
@Chr1st1anSears PR updated.

**Review by Chr1st1anSears — APPROVED** [review]

**Comment by Chr1st1anSears** [comment]
Thanks for your PR @Roming22--merging now :+1:

**Pull request merged**